Blockzilla/BrowserViewController.swift

```diff
@@ -467,35 +467,36 @@ class BrowserViewController: UIViewController {
         }
         
         // Screenshot the browser, showing the screenshot on top.
-        let image = mainContainerView.screenshot()
-        let screenshotView = UIImageView(image: image)
+        guard let screenshotView = view.snapshotView(afterScreenUpdates: true) else {
+            return
```

Comment:
Have you investigated when this returns nil? I'm not sure what the implication is here of returning without actually clearing the browser, and I don't want to introduce a case where users aren't able to clear (since it wasn't optional before). Is there a safer alternative? (ie. maybe just clearing the browser without the animation?). Might not be a big deal if we're confident that the snapshot won't return nil.

Reply:
The [documentation on the method](https://developer.apple.com/documentation/uikit/uiview/1622531-snapshotview) doesn't appear to explain when it's nil. I do agree that our fail case should be safer, though, and I've updated it accordingly!